packages/react-widgets/src/utils/useCustomCompareMemo.js

```diff
@@ -0,0 +1,11 @@
+import { useRef } from 'react';
+
+export default function useCustomCompareMemo(value, isEqual) {
```

Comment:
According to the rest of the project, hooks should be in a `hooks` folder so you should rename `utils` by `hooks`

Reply:
Done.